chef_solo_cup/helpers.py

```diff
@@ -40,6 +40,7 @@ def get_hosts(args, logger=None):
 
     for host, data in all_hosts:
         f = data.get('file', '')
+        g = data.get('group_name', '')
```

Comment:
Why not do something like:

`f = data.get('group_name', data.get('file', ''))`